templates/analytics/activity.html

```diff
@@ -9,7 +9,8 @@
 
 {% block customhead %}
 {{ super() }}
-{{ render_bundle('activity') }}
+{{ render_bundle('activity', 'css') }}
+{{ render_bundle('activity', 'js', attrs='defer') }}
```

Comment:
We should look at whether there's a way to set `attrs='defer'` in the defaults for the django webpack library rather than in every place we use it.

Reply:
There’s not. We could imagine having each template set a variable indicating which bundle(s) to render so that the code only needs to be written once in the base template, if you think that would be an improvement.

Reply:
Probably not worth that sort of restructuring just to avoid passing this option.  If we cared a lot, it's probably a fairly small patch we could send upstream if we find this annoying longer-term.

Reply:
I guess a Jinja macro is a reasonable structure for this.